conformance/conformance.proto

```diff
@@ -51,6 +51,12 @@ option java_package = "com.google.protobuf.conformance";
 //   - running as a sub-process may be more tricky in unusual environments like
 //     iOS apps, where fork/stdin/stdout are not available.
 
+enum WireFormat {
```

Comment:
The enum itself is intended to specify just a format itself, independent of whether it is an input format or output format. conformance_test.cc uses it internally to represent an input format. For ConformanceRequest.requested_output_format the field name makes it clear that it's an output format.